packages/table/src/Table.js

```diff
@@ -166,6 +170,11 @@ Table.propTypes = {
    * Additional props that will be spread to the table's root element
    */
   tableSpreadProps: PropTypes.any,
+  /**
+   * Optional prop for group checkbox toggle, setCheckboxToggle
+   */
+  checkboxToggle: PropTypes.bool,
```

Comment:
can we add these props to the grouped meta? it does not look like we expose anything about grouped data on this top level of props so it might lead to some confusion. we should also have a comment block for each prop as this shows up in the storybook description field in the prop table.

Reply:
updated passed prop for group select all to clarify use case. also added comments of prop and expected data.